python-sdk/conftest.py

```diff
@@ -141,7 +141,14 @@ def database_table_fixture(request):
     database.populate_table_metadata(table)
     database.create_schema_if_needed(table.metadata.schema)
     if file:
-        database.load_file_to_table(file, table)
+        database.load_file_to_table(
+            file,
+            table,
+            databricks_options={
```

Comment:
So far, we've avoided naming arguments of generic methods with implementation-specific names. Would it make sense to call this: `native_support_kwargs` - similar to other methods?

Reply:
+1

Reply:
Sounds good

Reply:
@tatiana @kaxil actually maybe it might be good to keep this as databricks options? The base class allows for arbitrary **kwargs so this could only matter if it's a databricks job. So if you're using delta it will matter and if you're not using delta it won't matter.

Reply:
My vote would still be to have a consistent interface across our implementations and try to have a container for platform-specific arguments - but I'm open to be persuaded otherwise :)